src/commands/deployment/gate.ts

```diff
@@ -91,7 +93,8 @@ export class DeploymentGateCommand extends Command {
   }
 
   private logger: Logger = new Logger((s: string) => this.context.stdout.write(s), LogLevel.INFO)
-  private pollingInterval = 15000
+  private evaluationRequestTimeout = 60000 // 1 minute
```

Comment:
We want to have a smaller timeout to request an evaluation. If we can't request it in 1 minute, we stop immediately. If we can request it but it's running, we wait until the user-specified timeout is reached